include/zephyr/drivers/sensor.h

```diff
@@ -246,6 +246,12 @@ enum sensor_trigger_type {
 
 	/** Trigger fires when no motion has been detected for a while. */
 	SENSOR_TRIG_STATIONARY,
+
+	/** Trigger fires when the FIFO threshold has been reached. */
+	SENSOR_TRIG_FIFO_THRESHOLD,
```

Comment:
Replace "threshold" with "watermark" to be consistent with the attribute enum?

Reply:
Done